libparsec/crates/platform_device_loader/src/lib.rs

```diff
@@ -416,3 +472,14 @@ fn secret_key_from_password(
         ),
     }
 }
+
+#[cfg_attr(target_arch = "wasm32", expect(dead_code))]
+fn server_url_from_device(device: &LocalDevice) -> String {
```

Comment:
Now that this is has been extracted we should add some unit tests for the function.

Reply:
done